src/components/dialogs/network-modifications/battery/creation/battery-creation-dialog.js

```diff
@@ -129,9 +128,9 @@ const BatteryCreationDialog = ({
                     ? 'MINMAX'
                     : 'CURVE',
                 minimumReactivePower:
-                    battery?.minMaxReactiveLimits?.minimumReactivePower ?? null,
```

Comment:
Why don’t you change the form fields like `minimumReactivePower` just above or maximumReactivePower below while you’re at it? I think it would make the code clearer.